### ISSUE TITLE
Include license text in its own file

### ISSUE THREAD

**Issue by jorti** [terencehonles/fusepy#59]
I'm packaging this module for Fedora and I see that you don't include the license text in its own file. Our policy [1] is to notify upstream about this and encourage you to add it.

In the source code files you could leave the copyright notice and a note of the license used and where is the license text.

I could make a pull request if you wish.

Kind regards.

[1] https://fedoraproject.org/wiki/Packaging:LicensingGuidelines?rd=Packaging/LicensingGuidelines#License_Text

**Comment by kaueraal** [comment]
This should really be done. Currently several files (e.g. examples/loopback.py) do not contain any license whatsoever. I suppose that isn't done on purpose, but it hinders me to publish a small script base on the file above under a open source license.

**Comment by terencehonles** [comment]
@kaueraal do the examples need licenses themselves or do the package license not cover it?

**Comment by kaueraal** [comment]
No, there simply wasn't a license for these files. Other files, e.g. `fuse.py` did contain one.
